libraries-bytecode/pom.xml

```diff
@@ -40,6 +40,21 @@
             <artifactId>asm-util</artifactId>
             <version>${asm.version}</version>
         </dependency>
+
+<!--        <dependency>-->
```

Comment:
Maybe we can put a comment above here to explain what to uncomment etc.

Reply:
Understood. Added